SMLHelper/Handlers/PingHandler.cs

```diff
@@ -0,0 +1,54 @@
+﻿using SMLHelper.V2.Patchers;
+using UnityEngine;
+
+namespace SMLHelper.V2.Handlers
+{
+    using Interfaces;
+
+    /// <summary>
+    /// A handler related to PingTypes
+    /// </summary>
+    public class PingHandler : IPingHandler
+    {
+        /// <summary>
+        /// Main entry point for all calls to this handler.
+        /// </summary>
+        public static IPingHandler Main { get; } = new PingHandler();
+
```

Comment:
@AlexejheroYTB @MrPurple6411  
Are we okay with not having static methods on this new handler?